lib/react.dart

```diff
@@ -384,7 +384,7 @@ class SyntheticFormEvent extends SyntheticEvent {
 class SyntheticDataTransfer {
   final String dropEffect;
   final String effectAllowed;
-  final List/*<File>*/ files;
+  final List files;
```

Comment:
In Strong mode you can add parameterized types via comments. And this was causing other issues if left typed.